### DISCUSSION TITLE
At runtime, there is a path error.

### DISCUSSION THREAD

**Discussion by yxjdbly** [NCAR/noahmp#133] [Q&A]
Dear professor,
 I added a new parameter in create_point_data.f90 and successfully generated hrldas_setup_single_point.nc and the forcing data files. However, I encountered an error during runtime. The error message is: "../HRLDAS_forcing/run/examples/single_point/hrldas_setup_single_point.nc". I tried using absolute paths but the issue persists. Could you please advise on the possible reasons for this?
![image](https://github.com/NCAR/noahmp/assets/168068037/b9084603-7dcc-4678-97e6-329962bdcc19)

**Comment by cenlinhe** [comment]
The error message seems to say that the data file path is wrong. Why do you have "x A" before "../HRLDAS_forcing/" in your path? Or are you able to access the file by "ncview ../HRLDAS_forcing/run/examples/single_point/hrldas_setup_single_point.nc"? Sometimes it may be also the file access issue.
